analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -190,15 +192,38 @@ class SpdxDocumentFile(
             Package(
                 id = pkg.toIdentifier(),
                 declaredLicenses = sortedSetOf(pkg.licenseDeclared),
-                concludedLicense = pkg.licenseConcluded.toSpdx(),
+                concludedLicense = buildConcludedLicense(pkg),
                 description = packageDescription,
                 homepageUrl = pkg.homepage.mapNotPresentToEmpty(),
                 binaryArtifact = RemoteArtifact.EMPTY, // TODO: Use "downloadLocation" or "externalRefs"?
-                sourceArtifact = RemoteArtifact.EMPTY, // TODO: Use "sourceInfo" or "externalRefs"?
+                sourceArtifact = buildSourceArtifact(pkg), // TODO: Use "sourceInfo" or "externalRefs"?
                 vcs = VersionControlSystem.forDirectory(packageDir)?.getInfo() ?: VcsInfo.EMPTY
             )
         }
 
         return listOf(ProjectAnalyzerResult(project, packages))
     }
+
+    /**
+     * Return null in case of "NONE" or "NOASSERTION" to fit with the standard process ( concluded manage by curations )
+     */
+    private fun buildConcludedLicense(pkg: SpdxPackage): SpdxExpression? {
+        return when {
+            pkg.licenseConcluded == SpdxConstants.NONE -> null
+            pkg.licenseConcluded == SpdxConstants.NOASSERTION -> null
+            else -> pkg.licenseConcluded.toSpdx()
+        }
+    }
```

Comment:
You could shorten the whole function to

    pkg.licenseConcluded.takeIf { SpdxConstants.isPresent(it) }?.toSpdx()

and just inline it into the caller.